test/cypress/features/delegates/delegates.js

```diff
@@ -9,6 +9,7 @@ import urls from '../../../constants/urls';
 const txConfirmationTimeout = 20000;
 
 Then(/^I see (\d+) delegates on page$/, function (number) {
+  cy.wait(100);
```

Comment:
@osvaldovega Any idea why is this wait needed and it wasn't needed before?
@Efefefef any idea for how to not need the wait?

Same for the other 3 `cy.wait(100);` in https://github.com/LiskHQ/lisk-hub/pull/2281/files